### PR TITLE
TNO-2764 duplicate summaries partial fix

### PR DESCRIPTION
**Pull request by nehalaggarwal-bcgov**
TNO-2764 duplicate summaries partial fix just shows summary if it exists.

### CODE REVIEW

**Review by areyeslo — COMMENTED** [review]
Can we remove the comments or Do we need them?

File: app/subscriber/src/features/content/view-content/ViewContent.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ import { ToneValue } from './ToneValue';
 import { isWorkOrderStatus } from './utils';
 
 //Difference ratio
-const threshold = 0.1;
+//const threshold = 0.1;
```
Comment (jdtoombs) [inline app/subscriber/src/features/content/view-content/ViewContent.tsx:35]: Looks good - Quick question, do we still need the commented out code? Here and below?

Comment (nehalaggarwal-bcgov) [reply app/subscriber/src/features/content/view-content/ViewContent.tsx:35]: Yes, I will be working on these later on to check if the summary is different from transcript.

**Review by areyeslo — APPROVED** [review]

File: app/subscriber/src/features/content/view-content/ViewContent.tsx (hunk 1 above)
Comment (areyeslo) [reply app/subscriber/src/features/content/view-content/ViewContent.tsx:35]: You'll need to remove them in the following PR.

**Pull request merged**